src/components/Nav.tsx

```
@@ -0,0 +1,31 @@
+  ];
+  return (
+    <div className="fixed bottom-0 flex justify-center w-full max-w-5xl bg-gradient-to-t to-transparent via-[#111] from-[#111] z-10">
+      <div className="flex justify-between w-full max-w-md">
+        {nav.map(({ icon, text, link }) => (
+          <div
+            onClick={() => navigate(link)}
+            className={`flex flex-col items-center gap-1 px-6 py-4 text-white cursor-pointer ${
+              location.pathname == link ? "text-white" : "text-zinc-500"
+            }`}
+          >
+            {icon}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)